opennms-provision/opennms-provision-persistence/src/main/java/org/opennms/netmgt/provision/persist/StringIntervalAdapter.java

```diff
@@ -57,10 +57,13 @@ public String marshal(final Duration v) {
     /** {@inheritDoc} */
     @Override
     public Duration unmarshal(final String v) {
-        if ("0".equals(v)) {
+        if ("0".equals(v.trim())) {
```

Comment:
Could `v` ever be null? This would introduce an NPE if so...

Reply:
null-check added